build/resolver/git/entrypoint.sh

```diff
@@ -157,7 +170,7 @@ wrapPull() {
     # Write commit id to output file, which will be collected by Cyclone
     cd $WORKDIR/data
     echo "Collect commit id to result file /__result__ ..."
-    echo "LastCommitID:`git log -n 1 --pretty=format:"%H"`" >> /__result__;
+    echo "LastCommitID:`git log -n 1 --pretty=format:"%H"`" > /__result__;
```

Comment:
Why do we need to change this?

Reply:
Don't need to append commit info to __result__. Just write to it is enough.